styles/style.css

```diff
@@ -10,9 +10,214 @@ img {
 body {
   font-size: 1rem;
   margin: 0;
+
 }
 
 ul {
   list-style: none;
   padding-left: 0;
 }
+
+
+.overwatch {
+  background: url('https://cdn2.pcadvisor.co.uk/cmsdata/reviews/3642282/overwatch-gameplay.jpg');
+  min-height: 100vh;
+}
+
+.hud {
+
+  position:fixed;
+  top: 72%;
```

Comment:
When using position absolute or fixed, push away from the closest edge unless you have a reason to do something like pushing past the outside of an element. Since you are pushing from the top to get the UI to look like it is near the bottom edge, this is guessing and breaks at different screen sizes since the content size remains the same. Instead use `bottom` and use an absolute unit instead of a percentage.

<img width="1280" alt="screen shot 2017-02-10 at 11 07 36 am" src="https://cloud.githubusercontent.com/assets/2532004/22836379/c4e8c6be-ef81-11e6-86e9-d4f44935e150.png">